ext/php_cassandra.c

```diff
@@ -173,12 +174,67 @@ ZEND_GET_MODULE(cassandra)
 #endif
 
 static void
-php_cassandra_log(const CassLogMessage* message, void *data)
+php_cassandra_log(const CassLogMessage* message, void* data)
 {
-  fprintf(stderr, "php-driver | [%s] %s (%s:%d)\n",
-    cass_log_level_string(message->severity), message->message,
-    message->file, message->line
-  );
+  char log[MAXPATHLEN + 1];
+  uint log_length = 0;
+
+  (void)data;
+
+  uv_rwlock_rdlock(&CASSANDRA_G(log_lock));
```

Comment:
looks like this fails in zts environment, because we're missing `TSRM_CC` in this context, we should use `TSRMLS_SET_CTX` and `TSRMLS_FETCH_FROM_CTX` to pass the thread context via `data` argument